### ISSUE TITLE
Export DepsTreeItem and DepsFlatItem in importable package.

### ISSUE THREAD

**Issue by jortel** [konveyor/analyzer-lsp#214]
Please export `DepsTreeItem` and `DepsFlatItem` from an importable package.

Recommend defining in or _aliasing_ in hubapi?

```
import (
   dependency "github.com/konveyor/analyzer-lsp/cmd/dep"
)
```
```
builder/deps.go:4:2: import "github.com/konveyor/analyzer-lsp/cmd/dep" is a program, not an importable package
```

Related (but not a blocker): recommend renaming `hubapi/` to something like: `types`, `report`, etc since the package seems to contain types produced by the analyzer and nothing to do with the Hub API.

**Comment by shawn-hurley** [comment]
Had a call with Jeff, and I am working on moving the types around now. 

With this said, I would like to add some commentary on the `hubapi` package name. Initially, the thought process from my perspective, was that there may be other output formats in the future (think junit for prow integration, or maybe some other type of output like LSP). the goal was to note that the types there was a contract between the hub addon and the output. 

Jeff brought up a good point, which is, if we have to talk to folks on how they are going to integrate into our tool, it may be confusing to say use the Hub API. while we mean the package, it could easily be thought that we are actually saying you can only use this through the hub API, which we do not want to be the case. 

Do we see any other default output for the tool? I don't believe so, so I think we should consider making it clear that this is the default output. I would consider `report` or `output` or maybe something like `output/v1/report` but that feels like maybe too much future proofing. 

@pranavgaikwad @fabianvf can y'all weigh in?

@jortel does this accuratly describe the concerns and what we talked about?

**Comment by pranavgaikwad** [comment]
@shawn-hurley thank you for following up on this. I like `output` as we are not outputting a report per se. I do not think there is any other potential output format, if there is, it probably should build up on our output format.

**Comment by fabianvf** [comment]
@shawn-hurley that's actually a great point about the name, and honestly I think adding a version to the output package early is also not a bad idea. It's just used on imports so it's not like it's crazy burdensome

**Comment by pranavgaikwad** [comment]
Fixed via #218
